src/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const compareArrays = (a: any[], b: any[]) => {
+    return JSON.stringify(a) === JSON.stringify(b);
```

Comment:
should these arrays be sorted first? or will it work if they have the same things but in diff order?

Reply:
ill add a sort option i guess